typescript/sdk/src/token/cosmosnativeDeploy.ts

```diff
@@ -0,0 +1,108 @@
+import { Logger } from 'pino';
+
+import { SigningHyperlaneModuleClient } from '@hyperlane-xyz/cosmos-sdk';
+import {
+  Address,
+  ProtocolType,
+  assert,
+  objFilter,
+  objMap,
+  rootLogger,
+} from '@hyperlane-xyz/utils';
+
+import { ChainMetadataManager } from '../metadata/ChainMetadataManager.js';
+import { ChainMap, ChainName } from '../types.js';
+
+import { TokenType, gasOverhead } from './config.js';
+import { WarpRouteDeployConfigMailboxRequired } from './types.js';
+
+export class CosmosNativeDeployer {
```

Comment:
Why is this class defined but not used anywhere? In the Module i see that the signer is used directly for deploying the tokens

Reply:
it is the cosmos native equivalent class to https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/chore/cosmos-sdk-warp-deploy/typescript/sdk/src/token/deploy.ts#L107 and gets only directly used in the cli later